src/device-registry/routes/v1/devices.js

```diff
@@ -443,21 +443,15 @@ router.post(
       )
       .bail()
       .notEmpty()
-      .withMessage("the name should not be empty if provided")
-      .bail()
-      .matches(/^[a-zA-Z0-9]+$/)
```

Comment:
no need to do input validations for` long_name` and `name` during device creation